src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="field">
```

Comment:
closing brace should be at the same level as opening one - this way your layout is better structured and it is easier to distinguish where ends one tag and the other starts
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      class="field"
    >
```

Reply:
fixed